dali/base/dafdesc.cpp

```diff
@@ -1282,7 +1285,7 @@ class CFileDescriptor:  public CFileDescriptorBase, implements ISuperFileDescrip
                 setReplicateFilename(fullpath,queryDrive(idx,copy),baseDir.str(),repDir.str());
 
                 const char *planeName = cluster->queryGroupName();
-                if (!isEmptyString(planeName))
+                if (!isEmptyString(planeName) || remoteStoragePlane)
```

Comment:
Was this wrong before and not specific to the group name ?

Reply:
If it's a foreign file it won't have the IClusterInfo's won't have resolved plane (aka group) names because those remote groups don't exist in the client environment, but if this is BM, and the foreign file is from k8s, there will be a remoteStoragePlane serialized/set, and it needs to enter this block to correctly stripe the files

Reply:
It's necessary to be part of this PR, because the suppression of the __anon groups exposes this issue. Before the suppression an "__anon*" group was created every time, and associated with the IClusterInfo's, meaning the planeName was not blank..

Reply:
I'll add. Also, this code should also apply in containerized, i.e. if there's a remoteStoragePlane it should use it (even in containerized). That covers the case where one k8s system reaches out to another k8s using foreign. That should not happen (for security reasons) - but the code makes sense to cover that case too.